lib/Twig/Extension/Core.php

```diff
@@ -537,26 +537,26 @@ function twig_urlencode_filter($url)
  *  {# items now contains { 'apple': 'fruit', 'orange': 'fruit', 'peugeot': 'car' } #}
  * </pre>
  *
- * @param array|Traversable $arr1 An array
- * @param array|Traversable $arr2 An array
+ * @param $arguments An array of array|Traversable which will be used to call array_merge.
  *
  * @return array The merged array
  */
-function twig_array_merge($arr1, $arr2)
+function twig_array_merge(...$arguments)
```

Comment:
This is not a good idea. It is breaking support for named arguments in Twig. You should keep the first 2 arguments